src/components/LibraryCardForm/LibraryCardForm.tsx

```diff
@@ -431,23 +442,12 @@ class LibraryCardForm extends React.Component<{}, LibraryCardFormState> {
           onBlur={this.handleOnBlur("email")}
           childRef={this.email}
         />
-        <FormField
```

Comment:
Previously, the custom `FormField` component use to render an `input` as either a text or checkbox type. But now the checkbox-related code branch is removed from there and simply using the `Checkbox` component.